hbase_regionserver/metadata.csv

```diff
@@ -7,9 +7,9 @@ hbase.regionserver.ipc.total_call_time.mean,gauge,"",millisecond,"","total call
 hbase.regionserver.ipc.total_call_time.median,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.ipc.total_call_time.percentile.99,gauge,"",millisecond,"","total call time, including both queued and processing time.",0,hbase_regionserver,""
 hbase.regionserver.regions.num_regions,gauge,"","","",Number of regions in the metrics system,0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_ops,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","","",0,hbase_regionserver,""
-hbase.regionserver.replication.sink.applied_batches,gauge,"","","","",0,hbase_regionserver,""
+hbase.regionserver.replication.sink.applied_ops,gauge,"","","",Number of WAL entries applied on replication sink.,0,hbase_regionserver,""
+hbase.regionserver.replication.sink.age_of_last_applied_op,gauge,"","","",Time lag of last applied WAL entry between master and slave in milliseconds.,0,hbase_regionserver,""
```

Comment:
I think there is no convention that "age" means miliis. I just updated description as time_unit is automatically set based on the word "time".